packages/hooks/src/stories/UseTokenBalance.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import { Provider, useWallet } from '..';
+import { useTokenBalance } from '../hooks/useTokenBalance';
+
+const Default = () => {
+  const { connectWallet, connected } = useWallet();
+
+  if (connected) {
+    return (
+      <>
+        <BalanceOfDai />
+        <UnknownOwner />
+        <UnknownContract />
+      </>
+    );
+  }
+
+  return <button onClick={connectWallet}>Connect Wallet</button>;
+};
+
+storiesOf('useTokenBalance', module).add('Default', () => (
+  <Provider network='mainnet'>
+    <Default />
+  </Provider>
+));
+
+const BalanceOfDai = () => {
+  const owner = '0x503828976D22510aad0201ac7EC88293211D23Da';
+  const address = '0x6B175474E89094C44Da98b954EedeAC495271d0F';
```

Comment:
can we pull these strings into constants at the top of the file (since they're reused) and name them appropriately? I assume that `address` is `DAI_CONTRACT_ADDRESS`?